### ISSUE TITLE
Decorative image using the aria-hidden=true instead of "alt" attribute

### ISSUE THREAD

**Issue by sarasuri** [w3c/wcag#3023]
Hi,
I have a question on decorative images on a Web page and meeting WCAG requirements.

A Web page has the following code for a decorative image. In the code below, the <img> element doesn’t use the “alt” attribute with null value instead using the “aria-hidden=true” to make the image not available to assistive technologies like screen readers.

`<img src="spacer.gif" width="100" height="1" aria-hidden="true">`

Would this approach acceptable to meet the SC 1.1.1 for decorative images?

**Comment by detlevhfischer** [comment]
I'd say yes, this if fine. It may not validate (I think alt is still required for img by the spec) without causing a 4.1.1 failure (and 4.1.1 is going to disappear soon anyway) but this is not an accessibility issue.

**Comment by stevefaulkner** [comment]
This may be helpful: [ALT TAG EMPTINESS](https://html5accessibility.com/stuff/2022/01/10/alt-tag-emptiness/)

**Comment by alastc** [comment]
@sarasuri I don't think you'll get a different answer, so I'll close this issue as answered.

**Comment by sarasuri** [comment]
Thank you all
@stevefaulkner, that is a great article with thorough explanation.
